lib/airbrake/shoryuken/error_handler.rb

```diff
@@ -0,0 +1,34 @@
+module Airbrake
+  module Shoryuken
+    ##
+    # Provides integration with Shoryuken.
+    class ErrorHandler
+      def call(_worker, _queue, _sqs_msg, body)
+        yield
+      rescue => e
+        notify_airbrake(e, notice_context(body))
+        raise e
+      end
+
+      private
+
+      def notice_context(body)
+        body.is_a?(Array) ? { batch: body } : { body: body }
+      end
+
+      def notify_airbrake(exception, context)
+        return unless (notice = Airbrake.build_notice(exception, context))
+
```

Comment:
It would be good to set action/component. Example:
https://github.com/airbrake/airbrake/blob/c45bd6e0b4955dd0a0ad7fa87590383cc65fd8a4/lib/airbrake/sidekiq/error_handler.rb#L19

I'm not sure what the action would be in case of Shoryuken, so please let me know what you think suits best for this integration (you can check other integrations to see what we usually set).